setup.py

```diff
@@ -90,6 +90,7 @@ def setup(*args, **kwargs):
         "scipy>=1.1",
     ],
     tests_require=[
+        "ase==3.22.0",
```

Comment:
Given the issues that CI is having with ASE, is there a lower ASE version we can use here?

Reply:
Lowered the version of ASE to 3.21.1 which seems to have worked out fine!